src/emqx_protocol.erl

```diff
@@ -474,7 +496,14 @@ deliver({connack, ReasonCode}, PState) ->
 deliver({connack, ?RC_SUCCESS, SP}, PState = #pstate{zone = Zone,
                                                      proto_ver = ?MQTT_PROTO_V5,
                                                      client_id = ClientId,
+                                                     conn_props = CONNPROPS,
                                                      is_assigned = IsAssigned}) ->
+    case maps:find('Request-Response-Information', CONNPROPS) of
```

Comment:
nit: avoid binding variables in case clauses and use them outside.
bind the case expression result to variable instead.
i.e
```
ResponseInfomation = case maps:find(....) of
    {ok, 1} -> ClientId;
    _ -> <<>>
end.
```

Reply:
Yes, you are right. :)